environment.py

```diff
@@ -912,5 +912,13 @@ def env() -> Mapping[str, Optional[str]]:
         # in the Azul VPC. This subnet can't overlap the VPC CIDR and the subnet
         # mask must be less than 22 bits.
         #
-        'azul_vpn_subnet': None
+        'azul_vpn_subnet': None,
+
+        # A global rate limit on file downloads across all regions and IP
+        # addresses, evaluated at 10-minute intervals and enforced by AWS WAF.
+        # The value must also include the expected number of distinct IPs making
+        # at least one download request per 10-minute window, separated by a `/`
+        # (e.g. "100/3.5"; the concurrency does not need to be an integer).
```

Comment:
Let's do 

```suggestion
        # (e.g. "100/600@3.5"; the concurrency does not need to be an integer).
```

where 100 is the numerator of the rate, 600 is the denominator in seconds (mention that 600 is the minimum) and 3.5 is the assumed concurrency.